### ISSUE TITLE
What's the use of the <GUID> archive folder after cleanup?

### ISSUE THREAD

**Issue by TobiasKnauss** [MicrosoftDocs/visualstudio-docs#5285]
As far as I understand, you only need the subfolders inside the "archive" folder for cleanup.
a) If you delete these folders without cleanup, then a subsequent cleanup is not possible anymore.
b) If you perform a cleanup, then these folders become useless, except for documentary purposes.
Are my assumptions correct?
If yes, this means that I can delete these folders after a successful cleanup.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dc51a257-b7a9-4660-7e12-88cdfcd9cd53
* Version Independent ID: dad2fd97-7244-6802-e853-5d5974838996
* Content: [Update a network-based installation - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/install/update-a-network-installation-of-visual-studio?view=vs-2019#feedback)
* Content Source: [docs/install/update-a-network-installation-of-visual-studio.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/install/update-a-network-installation-of-visual-studio.md)
* Product: **visual-studio-windows**
* Technology: **vs-installation**
* GitHub Login: @ornellaalt
* Microsoft Alias: **ornella**

**Comment by WilliamAntonRohm** [comment]
@TobiasKnauss -- thank you for your question. Here are a few options you might consider:​  

- [https://social.msdn.microsoft.com/Forums/vstudio](https://social.msdn.microsoft.com/Forums/vstudio) 
- [https://www.stackoverflow.com](https://www.stackoverflow.com) 
- [https://www.reddit.com/r/VisualStudio/](https://www.reddit.com/r/VisualStudio) 

@ornellaalt -- please look into this issue.

**Comment by ornellaalt** [comment]
@mjcavallari can you please look into this issue?

**Comment by mjcavallari** [comment]
When a layout has multiple archive folders, it is assumed that the collection of catalogs covers the collection of packages in the layout.  When using the clean operation, the installer will enumerate the packages listed in the catalog version beign removed unless they show up in one of the catalogs being left.

With that in mind, I can confirm that if the catalog is removed before doing a cleanup, the installer is unable to remove those obsolete packages.  But once a cleanup is done, the archive folder is no longer needed.  We keep the archive so that users are able to validate their layout.  If it is found that the version is needed, the catalog is available to restore the version.  If you no longer need that archive, you are able to manually delete the archive folder.
